### PR TITLE
Chchchchanges

### PR DESCRIPTION
**Pull request by sillyfilly**
For James

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ teeeeest
 oomble moomble
 
 
-yusyusyusyusyus
```
Comment (James-Evans) [inline README.md:7]: shite

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ teeeeest
...
+Nononono!
```
Comment (James-Evans) [inline README.md]: yer having a larf

File: README.md (hunk 1 above)
Comment (sillyfilly) [inline README.md:7]: I beg to differ

File: README.md (hunk 2 above)
Comment (sillyfilly) [inline README.md]: Aye fair enough

Comment (sillyfilly) [inline README.md]: Changed it now

**Pull request merged**